tests/core/tests/test_widgets.py

```diff
@@ -609,8 +637,11 @@ def test_render(self):
             "%s,%s" % (self.cat1.name, self.cat2.name),
         )
 
+    def test_render_value_is_None(self):
```

Comment:
tiny nit: the name of this test jumped out at me as confusing. maybe 'test_render_value_none_as_blank' or something

Reply:
Good call - have updated.